frontend/components/board.js

```diff
@@ -0,0 +1,167 @@
+var React = require('react')
+
+function Square(props) {
+  return (
+    <button className="square" id={props.value}>
+    </button>
+  );
+}
+
+module.exports = React.createClass({
+  getInitialState: function() {
+    var squares = Array(100).fill(null);
+    var isBlack = true;
+    for(var i=0; i<squares.length; i++) {
+      if(i%10 == 0) {
+        isBlack = !isBlack
+      }
+
+      if(isBlack) {
+        squares[i] = "black";
+        isBlack = !isBlack;
+        continue;
+      }
+
+      squares[i] = "white";
+      isBlack = !isBlack;
+    }
+    return {
+      squares: squares,
+    }
+  },
+
+  renderSquare: function(i) {
+    return (
+      <Square
+        value={this.state.squares[i]}
+      />
+    );
+  },
+
+  render: function() {
+    return (
+      <div>
+        <div className="board-row">
```

Comment:
Can you do these in loops?

Reply:
I'll add that in the next commit. I initially didn't think it would be possible since the tutorial hardcoded in a similar way, but looking through StackOverflow it's definitely doable